tools/gulpfile.js

```diff
@@ -232,6 +239,7 @@ gulp.task('update-react-native-admob', () => {
     targetAndroidPath: 'modules/api/components/admob',
     sourceAndroidPackage: 'com.sbugert.rnadmob',
     targetAndroidPackage: 'versioned.host.exp.exponent.modules.api.components.admob',
+    installableInManagedApps: false,
```

Comment:
shouldn't this be true?

Reply:
There is `expo-ads-admob`, I assumed installing that package (as suggested by our docs) would be the way to use this module in Expo apps.